r/src/filesystem.cpp

```diff
@@ -422,4 +424,63 @@ std::shared_ptr<fs::GcsFileSystem> fs___GcsFileSystem__Make(bool anonymous,
   return fs::GcsFileSystem::Make(gcs_opts, io_context);
 }
 
+// [[gcs::export]]
+cpp11::list fs___GcsFileSystem__options(const std::shared_ptr<fs::GcsFileSystem>& fs) {
+  using cpp11::literals::operator"" _nm;
+
+  cpp11::writable::list out;
+
+  fs::GcsOptions opts = fs->options();
+
+  // GcsCredentials
+  out.push_back({"anonymous"_nm = opts.credentials.anonymous() });
+
+  if (opts.credentials.access_token() != "") {
+    out.push_back({"access_token"_nm = opts.credentials.access_token() });
+  }
+
+  // TODO: Cast to posixct?
```

Comment:
This needs more work. What I'd like is to get this back to a POSIXct but I'm not sure how to do that.